tslearn/shapelets.py

```diff
@@ -248,54 +276,50 @@ class ShapeletModel(BaseEstimator, ClassifierMixin):
     ----------
     .. [1] J. Grabocka et al. Learning Time-Series Shapelets. SIGKDD 2014.
     """
-    def __init__(self, n_shapelets_per_size,
-                 max_iter=1000,
+    def __init__(self, n_shapelets_per_size=None,
+                 max_iter=100,
                  batch_size=256,
-                 verbose_level=2,
+                 verbose_level=0,
                  optimizer="sgd",
                  weight_regularizer=0.,
+                 shap_len=0.15,
```

Comment:
We could rename that to `shapelet_length` to make it more explicit?

Similarly below for `nr_shap_lens`, a name with no abbreviation would be welcome